README.md

```diff
@@ -101,6 +101,18 @@ pip install alogamous
       ```bash
       sudo chmod a+r /usr/local/hatch/bin/hatch
       ```
+### Adding to Gitignore
```

Comment:
This is a great addition to the README.

Do you think you could add a section on how to add lines to the `.gitignore` file and what they mean? You could pull a few example from the [git docs](https://git-scm.com/docs/gitignore)